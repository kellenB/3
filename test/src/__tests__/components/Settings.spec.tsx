import React from 'react'
import { render } from '@testing-library/react'

import Settings from '../../components/Settings'

describe('Settings page', () => {
  it('Matches snapshot', () => {
    const component = render(<Settings onSubmit={jest.fn()} />)

    expect(component).toMatchSnapshot()
  })
})