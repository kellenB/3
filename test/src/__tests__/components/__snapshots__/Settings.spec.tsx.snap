// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings page Matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="Settings"
      >
        <form>
          <h1>
            Settings
          </h1>
          <h2>
            Email time
          </h2>
          <label>
            Start time
          </label>
          <input
            max="00:00"
            type="time"
            value="00:00"
          />
          <label>
            End time
          </label>
          <input
            min="00:00"
            type="time"
            value="00:00"
          />
          <h2>
            Colours
          </h2>
          <label>
            Fill Color
          </label>
          <span>
            <input
              type="color"
              value="#57a5af"
            />
            <i
              class="material-icons"
            >
              clear
            </i>
          </span>
          <label>
            Highlight Color
          </label>
          <span>
            <input
              type="color"
              value="#5eb685"
            />
            <i
              class="material-icons"
            >
              clear
            </i>
          </span>
          <label>
            Background Color
          </label>
          <span>
            <input
              type="color"
              value="#efefef"
            />
            <i
              class="material-icons"
            >
              clear
            </i>
          </span>
          <button
            class="Button"
            type="submit"
          >
            Save Settings
          </button>
        </form>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="Settings"
    >
      <form>
        <h1>
          Settings
        </h1>
        <h2>
          Email time
        </h2>
        <label>
          Start time
        </label>
        <input
          max="00:00"
          type="time"
          value="00:00"
        />
        <label>
          End time
        </label>
        <input
          min="00:00"
          type="time"
          value="00:00"
        />
        <h2>
          Colours
        </h2>
        <label>
          Fill Color
        </label>
        <span>
          <input
            type="color"
            value="#57a5af"
          />
          <i
            class="material-icons"
          >
            clear
          </i>
        </span>
        <label>
          Highlight Color
        </label>
        <span>
          <input
            type="color"
            value="#5eb685"
          />
          <i
            class="material-icons"
          >
            clear
          </i>
        </span>
        <label>
          Background Color
        </label>
        <span>
          <input
            type="color"
            value="#efefef"
          />
          <i
            class="material-icons"
          >
            clear
          </i>
        </span>
        <button
          class="Button"
          type="submit"
        >
          Save Settings
        </button>
      </form>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
